authors/apps/articles/views.py

```diff
@@ -677,3 +677,29 @@ def get_all_comments(slug, id):
         article = get_object_or_404(Article, slug=slug)
         highlight_comment = article.highlight.filter(id=id)
         return highlight_comment
+
+        
+class CommentHistory(ListCreateAPIView):
+
+    serializer_class = CommentHistorySerializer
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request, id, slug, **kwargs):
+
+        try:
+            # get an article that matches the slug specified
+            # and return a message if the article does not exist
+            Article.objects.get(slug=slug, is_deleted=False)
+        except ObjectDoesNotExist:
+            raise NotFound("This article does not exist")
+        id = self.kwargs['id']
```

Comment:
is this the same id with the one passed into the function? if it is, is this still required?